src/edu/ucsb/nceas/metacat/util/AuthUtil.java

```diff
@@ -475,5 +494,33 @@ private static boolean onAccessList(Vector<String> accessList, String username,
 		}
 		return false;
 	}
+	
+	/**
+     * Convert a delimited string list to a Vector
+     * @param list  the list of delimited vectors
+     * @param delimiter  the string to specify the delimiter
+     * @param escapeChar  the string to escape a delimiter.
+     * @return a vector holding the vales. An empty vector will be return if list is null or empty.
+     */
+    public static Vector<String> split(String list, String delimiter, String escapeChar) {
+        Vector<String> results = new Vector<String>();
+        logMetacat.debug("AuthUtil.split -the list is "+delimiter);
+        logMetacat.debug("AuthUtil.split - the delimiter is "+delimiter);
+        logMetacat.debug("AuthUtil.split -the escapeChar is "+ escapeChar);
```

Comment:
This debug output seems like overkill, plus you print the delimiter twice, rather than the list and delimiter.